descope/internal/auth/auth.go

```diff
@@ -894,7 +894,15 @@ func composeUpdateUserEmailEnchantedLink() string {
 }
 
 func composeOAuthURL() string {
```

Comment:
I think I would rename composeOAuthURL to composeOAuthSignUpOrInURL to keep it consistent